pkg/time/rate/rate.go

```diff
@@ -179,7 +179,6 @@ func (r *Reservation) DelayFrom(now time.Time) time.Duration {
 // Cancel is shorthand for CancelAt(time.Now()).
 func (r *Reservation) Cancel() {
 	r.CancelAt(time.Now())
```

Comment:
please don't touch this file.
It's a fork from standard golang library that we should sync from upstream from time to time. We forked it because we wanted some more specific feature used in our bucket system. We should even propose our changes upstream.

Reply:
done, I've reverted that